docs/source/executing-operations/subscription-support.mdx

```diff
@@ -98,11 +98,18 @@ flowchart LR;
 
 1. Update your Apollo Router instances to version `1.22.0` or later. [Download the latest version.](../quickstart/)
     - Previous versions of the Apollo Router don't support subscription operations.
-2. Make sure your router is [connected to a GraphOS Enterprise organization](../enterprise-features/#enabling-enterprise-features).
+1. Make sure your router is [connected to a GraphOS Enterprise organization](../enterprise-features/#enabling-enterprise-features).
     - Subscription support is an Enterprise feature of self-hosted routers.
-3. **If you compose your router's supergraph schema with GraphOS** (instead of with the Rover CLI), [update your build pipeline](/graphos/graphs/updating#2-update-your-build-pipeline) to use Apollo Federation 2.4 or later.
+1. **If you compose your router's supergraph schema with GraphOS** (instead of with the Rover CLI), [update your build pipeline](/graphos/graphs/updating#2-update-your-build-pipeline) to use Apollo Federation 2.4 or later.
     - Previous versions of Apollo Federation don't support subscription operations.
-4. Modify your subgraph schemas to use Apollo Federation 2.4 or later:
+1. Each package that makes up Apollo Federation needs to be updated to version 2.4 or later:
+    - `@apollo/gateway`
+    - `@apollo/query-planner`
+    - `@apollo/subgraph`
+    - `@apollo/federation-internals`
+    - `@apollo/query-graphs`
+    - `@apollo/composition`
```

Comment:
I think we should mention it's only related to npm packages if they use apollo server